EpiJS/comp.js

```diff
@@ -61,6 +61,8 @@ class Idiom {
     let out = math.evaluate(parsed.join(''))
 
     // Round up to 0 if below 0 for out
+		/* istanbul ignore if */
+		/* istanbul ignore if */
```

Comment:
Why is this being ignored?

Reply:
This cannot be assured to run every single time, so we ignore it.